libs/src/svkImageReader2.cc

```diff
@@ -616,7 +637,33 @@ void svkImageReader2::GlobFileNames()
     }
     
     if( groupToUse != -1 ) {
+
+        vtkStringArray* group = sortFileNames->GetNthGroup(groupToUse); 
+        int numFilesInGroup = group->GetNumberOfValues();
+        cout << "NUM VALUES: " <<  numFilesInGroup << endl;
+
+        //  Before setting the file names for this group, confirm that if more than
+        //  one file in the group that they all have the same series description
+        //  all have the same series description    
+        if ( numFilesInGroup > 1 ) {   
+            
+            string referenceSeriesDescription = this->GetFileSeriesDescription( fileName ); 
+            //cout << "REF SERIES DESCRIPTION: " << referenceSeriesDescription << endl; 
+      
+            vtkStringArray* seriesGroup = vtkStringArray::New();  
+            for (int i = 0; i < numFilesInGroup; i++) {
+                string groupFileName = sortFileNames->GetNthGroup( groupToUse )->GetValue(i); 
+                string seriesDescription = this->GetFileSeriesDescription( groupFileName ); 
+                //cout << "SERIES DESCRIPTION: " << seriesDescription << endl;
+                if ( seriesDescription.compare( referenceSeriesDescription ) == 0 ) {
+                    seriesGroup->InsertNextValue( groupFileName ); 
+                }
+            }
```

Comment:
Will this loop run for DICOM data as well as other formats, such as IDF's? If so will there be a big performance hit when loading DICOM series with a large number of files such as perfusion or diffusion?